Development/cpprest/host_utils.cpp

```diff
@@ -5,6 +5,7 @@
 #include <boost/asio/ip/host_name.hpp>
 #include <boost/asio/ip/tcp.hpp>
 #include <boost/range/adaptor/filtered.hpp>
+#include <boost/range/algorithm.hpp>
```

Comment:
If it wouldn't involve a lot of `#include`s, could you avoid the convenience header, and use the precise ones that are needed?

Maybe this?

```suggestion
#include <boost/range/algorithm/find.hpp>
#include <boost/range/algorithm/find_if.hpp>
```